### ISSUE TITLE
azurerm_servicebus_namespace is missing infrastructure encryption

### ISSUE THREAD

**Issue by kingofthehill444** [hashicorp/terraform-provider-azurerm#15630]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

azurerm_servicebus_namespace is missing infrastructure encryption even though it is available through the Service Bus ARM template and the Azure rest API.

### New or Affected Resource(s)

* azurerm_servicebus_namespace

### Potential Terraform Configuration

```hcl
###########################
# Service Bus Namespace
###########################
resource "azurerm_servicebus_namespace" "this" {
  name                              = "test-service-bus"
  location                          = var.location
  resource_group_name               = var.resource_group_name
  sku                               = var.sku
  capacity                          = var.capacity
  tags                              = var.tags
  infrastructure_encryption_enabled = var.infrastructure_encryption_enabled
}
```

**Comment by mybayern1974** [comment]
@kingofthehill444 would you mind I closing this issue given it's dup with #15629? I.e. you can subscribe to #15629

**Comment by mybayern1974** [comment]
My fault. They are different issues. Sorry and I've reopened this issue.

**Comment by catriona-m** [comment]
Thanks for opening this @kingofthehill444. It looks like this issue is now resolved in #15601, so this can now be closed.
